src/wayland/seat/keyboard/keymap_file.rs

```diff
@@ -77,14 +80,9 @@ impl SealedFile {
 
         file.seek(std::io::SeekFrom::Start(0))?;
 
-        nix::fcntl::fcntl(
+        fchmod(
             file.as_raw_fd(),
-            FcntlArg::F_ADD_SEALS(
-                SealFlag::F_SEAL_SEAL
-                    | SealFlag::F_SEAL_SHRINK
-                    | SealFlag::F_SEAL_GROW
-                    | SealFlag::F_SEAL_WRITE,
-            ),
+            Mode::S_IRUSR | Mode::S_IRGRP | Mode::S_IROTH
```

Comment:
Android will need a cfg guard, the sealing is intentional to prevent clients from messing up.

Reply:
reverted the change (since this PR uses nix 0.25 now)